pom.xml

```diff
@@ -86,19 +86,19 @@
     <dependency>
       <groupId>com.google.jimfs</groupId>
       <artifactId>jimfs</artifactId>
-      <version>1.0</version>
+      <version>1.1</version>
       <scope>test</scope>
     </dependency>
     <dependency>
       <groupId>org.mockito</groupId>
       <artifactId>mockito-core</artifactId>
-      <version>1.10.16</version>
+      <version>2.11.0</version>
       <scope>test</scope>
     </dependency>
     <dependency>
       <groupId>org.eclipse.jdt.core.compiler</groupId>
```

Comment:
Is this still used?

Reply:
Not sure, can check

Reply:
There's a single test in `FunSpecTest` that uses Mockito, maybe there's a way to rewrite it and get rid of the dependency